.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,14 @@
 blank_issues_enabled: false
 contact_links:
   - name: Gatsby Documentation
-    url: https://www.gatsbyjs.org/docs/
+    url: https://www.gatsbyjs.com/docs/
     about: Check out the Gatsby documentation for answers to common questions
   - name: Gatsby Discord
     url: https://gatsby.dev/discord
     about: Ask questions, get help and discuss new things you're building with the Gatsby community
   - name: Gatsby Discussions
     url: https://github.com/gatsbyjs/gatsby/discussions
```

Comment:
How about moving this up to the top, and calling it something like "Question, Help or Feature Request 🤔" to match the names of the issue templates

Reply:
The order is like this due to:

1. Self-Serve
2. More interactive chat
3. Slower chat
4. Our own Support

So ideally people solve their issues by going to the docs

Reply:
I'd agree if it weren't for the fact that the top item is always to open an issue. Encouraging some of those to use discussions instead would be a good thing.